pages/contact.vue

```diff
@@ -1,15 +1,109 @@
 <template>
-  <div class="white-wrapper">
-    Contact Page
+  <div class="background-section">
+    <div class="title">
+      <h1 class="cursive">Contact & Directions</h1>
+      <h4><font-awesome-icon icon="phone"/> (617) 924-9595</h4>
+    </div>
+    <div class="contact-section">
+        <div class="outside-image">
+          <img src="outside2.jpg">
+        </div>
+        <div class="store-info-section">
+          <contact-info />
+        </div>
+    </div>
+    <div class="call-section">
+      <div class="order">
+        <a :href="`tel:${6179249595}`">
+          <div class='phone-number'> 
+            <font-awesome-icon icon="phone"/> (617) 924-9595 
+          </div>
+        </a>
+      </div>
+      <a :href="`tel:${6179249595}`">
```

Comment:
This is awesome! Nice job man!

Reply:
Can you apply this to the `Order Now` part of the `ContactInfo.vue` component? I think we should have that button be clickable for a call as well.

Reply:
In process for ```ContactInfo.vue```. When I add the anchor tag to the Order Now button the button becomes half the size but the whole area of the original button remains clickable.